scripts/use_node

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
```

Comment:
Not recommended as ubuntu is using `ash` linking to `sh`.

Reply:
This is what OSD used before but I see that OS uses `#!/usr/bin/env bash`. Shall I use that?

Reply:
Ahhh, if that is the case better just keep `sh` as not all the docker images install `env` by default as well.
On debian I have already force a `#!/bin/bash` just to avoid.

Reply:
I really dont have too much solution to this generalized script tho.

Reply:
If OS is already using env, then we probably can just use.
At least env is more common than which not being installed.
Thanks.

Reply:
Thanks will `!/usr/bin/env bash`

Reply:
Spoke to Peter offline; will experiment with both options on Ubuntu and decide if we an retain the `#!/bin/sh` which is used by all OSD scripts.

Reply:
@peterzhuamazon I did some reading and most of what I found reasonable indicate a preference for using `#!/bin/sh`.
https://stackoverflow.com/a/55927235
https://unix.stackexchange.com/a/496523
https://www.ixsystems.com/blog/migrating-from-linux-to-freebsd/

Reply:
Thanks @AMoo-Miki !

I have no issue with that then.

Reply:
So it was intentional to not make this use_node.sh and set as bin/bash? If so should we migrate the other shell scripts to this format?

Reply:
@AMoo-Miki  any comment about the above ^

Reply:
We should standardize what we do. I will raise an issue for it.